### PR TITLE
Added ActionScript 3 implementation.

### PR DESCRIPTION
**Pull request by komlosizsolt**

### CODE REVIEW

File: ActionScript 3/BKKCryptTest.as [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+			
+		}
+		
+		public function TestEncrypt():Boolean 
+		{
+			var password:String = "password";
+			var bkk:BKKCrypt = new BKKCrypt();
+			if (bkk.Encrypt(password)==password) 
+			{
+				return true;
```
Comment (rolandhegedus) [inline ActionScript 3/BKKCryptTest.as]: We are not using returns in a test imho.

File: ActionScript 3/ICrypt.as [hunk 2]
```diff
@@ -0,0 +1,7 @@
+package 
+{
+	public interface ICrypt
```
Comment (rolandhegedus) [inline ActionScript 3/ICrypt.as]: There is no need for an interface. Should be refactored.

File: ActionScript 3/BKKCryptTest.as [hunk 3]
```diff
@@ -0,0 +1,24 @@
+package 
+{
+	public class BKKCryptTest 
+	{
+		
+		public function BKKCryptTest()
```
Comment (rolandhegedus) [inline ActionScript 3/BKKCryptTest.as]: Useless constructor method. Please refactor.

File: ActionScript 3/BKKCrypt.as [hunk 4]
```diff
@@ -0,0 +1,18 @@
+package 
+{
+
+	public class BKKCrypt implements ICrypt 
+	{
+		
+		public function BKKCrypt()
```
Comment (rolandhegedus) [inline ActionScript 3/BKKCrypt.as]: Useless constructor, please refactor.

File: ActionScript 3/BKKCryptTest.as (hunk 1 above)
Comment (ZackRave-N) [inline ActionScript 3/BKKCryptTest.as]: FlexUnit should be used

File: ActionScript 3/BKKCrypt.as [hunk 5]
```diff
@@ -0,0 +1,18 @@
...
+{
+
+	public class BKKCrypt implements ICrypt 
+	{
+		
+		public function BKKCrypt() 
+		{
+		}
+
+		public function Encrypt(input:String):String
```
Comment (ZackRave-N) [inline ActionScript 3/BKKCrypt.as]: Please look at Java implementation and use the same function and package names so the code will be the same in across languages

Comment (ZackRave-N) [inline ActionScript 3/BKKCrypt.as]: Also please add comments as described in the CONTRIBUTING.md

**Pull request merged**